homeassistant/components/device_tracker/nmap_tracker.py

```diff
@@ -102,10 +106,14 @@ def _update_info(self):
             if last_results:
                 # Pylint is confused here.
                 # pylint: disable=no-member
-                options += " --exclude {}".format(",".join(device.ip for device
-                                                           in last_results))
+                exclude_hosts = self.exclude + [device.ip for device
```

Comment:
The device tracker skips scanning for devices that were recently seen by adding them to the exclude list. I took the text formatting out of that section and moved it one step lower.